hazelcast/src/main/java/com/hazelcast/jet/impl/MasterJobContext.java

```diff
@@ -452,6 +454,10 @@ Tuple2<CompletableFuture<Void>, String> requestTermination(
             mode = mode.withoutTerminalSnapshot();
         }
 
+        if (!Util.checkJobIsAllowedToBeSuspended(mode, mc.jobConfig())) {
+            return tuple2(executionCompletionFuture, "Cannot suspend the job being analyzed");
```

Comment:
if we have general flag ("is suspendable"), error message also should be general, something like "job for which suspension is disabled" or you probably can come up with something better

Reply:
Renamed